ResidentInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -63,42 +63,46 @@ public static MosaicResidentInformationResponse ToResponse(this MosaicResidentIn
                 LastName = domain.LastName,
                 Uprn = domain.Uprn,
                 DateOfBirth = domain.DateOfBirth,
-                PhoneNumber = domain.PhoneNumberList?.ToResponse(),
+                PhoneNumber = domain.PhoneNumber?.ToResponse(),
                 AddressList = domain.AddressList?.ToResponse(),
                 NhsNumber = domain.NhsNumber
             };
         }
 
-        private static List<Phone> ToResponse(this List<PhoneNumber> phoneNumbers)
+        private static List<Boundary.Responses.Phone> ToResponse(this List<Phone> phoneNumbers)
         {
             return phoneNumbers.Select(r => r.ToResponse()).ToList();
         }
 
-        private static Phone ToResponse(this PhoneNumber domain)
+
+        private static Boundary.Responses.Phone ToResponse(this Phone phoneNumber)
         {
-            return new Phone
+            return new Boundary.Responses.Phone
             {
-                PhoneNumber = domain.Number,
-                PhoneType = domain.Type.ToResponse()
+                PhoneNumber = phoneNumber.PhoneNumber,
+                PhoneType = phoneNumber.PhoneType.ToResponse(),
             };
         }
 
-        private static PhoneTypeResponse ToResponse(this PhoneType domain)
+        private static PhoneTypeResponse ToResponse(this string domain)
         {
             switch (domain)
             {
-                case PhoneType.Primary:
+                case "Primary":
```

Comment:
In housing API, the phone types are single characters representing the phone type e.g P for Primary. 
Maybe this could be changed in the housing API to return them as the full descriptions instead for clarity?

Reply:
ah yeah, I got distracted by the enum descriptions 🤦‍♀️ . Do you know what X means in the enum?

Reply:
I have no clue 😂 , maybe unknown?

Reply:
I'll go with that :)